server/server.js

```diff
@@ -34,7 +33,31 @@ const pool = new pg.Pool({
   connectionString: "postgres://postgres:password@127.0.0.1:5432/pet_database"
 })
 
-// Express routes
+app.get("/", (req, res) => {
+  res.redirect("/pets")
```

Comment:
You have this redirect to /pets, though I do not see an endpoint for /pets. Requesting clarification, or a change on this.

Reply:
We have a "*" route that handles all incoming non-API routes, including /pets. Is that ok?

Reply:
It should be all good. I am assuming there will be/is a path in your React Router for /pets? If so that should be good.